lib/cli/src/extract.js

```diff
@@ -0,0 +1,111 @@
+import path from 'path';
+import { writeFile } from 'fs-extra';
+import puppeteerCore from 'puppeteer-core';
+import express from 'express';
+import getPort from 'get-port';
+import { logger } from '@storybook/node-logger';
+
+const read = async url => {
+  const puppeteer = await usePuppeteer();
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  await page.goto(url);
+
+  await page.waitForFunction(
+    'window.__STORYBOOK_STORY_STORE__ && window.__STORYBOOK_STORY_STORE__.extract && window.__STORYBOOK_STORY_STORE__.extract()'
+  );
+  const data = JSON.parse(
+    await page.evaluate(async () => {
+      // eslint-disable-next-line no-undef
+      const d = window.__STORYBOOK_STORY_STORE__.extract();
+
+      const result = Object.entries(d).reduce(
+        (acc, [k, v]) => ({
+          ...acc,
+          [k]: {
+            ...v,
+            parameters: {
```

Comment:
we might need to revisit this filtering of parameters in the future, but ok for now